src/scheduled-tasks-service.ts

```diff
@@ -67,3 +67,78 @@ export async function getTasksForFormsApp(
     }
   }
 }
+
+/**
+ * Complete the related Task for a specific Forms App
+ *
+ * #### Example
+ *
+ * ```js
+ * const formsAppId = 1
+ * const taskId = 2
+ * const completedTask = await scheduledTasksService.completeTask(
+ *   formsAppId,
+ *   taskId,
+ * )
+ * ```
+ *
+ * @param formsAppId
+ * @param taskId
+ * @param abortSignal
+ * @returns
+ */
+export async function completeTask(
+  formsAppId: number,
+  taskId: number,
```

Comment:
can we please make these two params an object as its easy to pass them in the incorrect order because they are both numbers